.make_ghpages/generate_json.py

```diff
@@ -51,9 +51,29 @@ def parse_yaml_files_in_folder(folder_path, exclude=None):
     link = os.readlink(library_path/domain/'default')
     final_dict[domain]['default'] = str(Path(link))
 
+# Prepare the config db for aiida 2.x data type entry points compatibility
+def update_to_v2_entry_points(comp_setup: dict) -> dict:
+    # v1 -> v2 with attach `core.` in front for transport and scheduler
+    res = copy.deepcopy(comp_setup)
+    for k, v in comp_setup.items():
+        if k in ['transport', 'scheduler']:
+            res[k] = '.'.join(['core', v])
+
+    return res
+
+final_dict_v2 = copy.deepcopy(final_dict)
+
+# Loop over or the fields and update to compatible with aiida 2.x entry points name
+for domain in final_dict_v2:
+    for computer in final_dict_v2[domain]:
+        if computer != 'default':
+            final_dict_v2[domain][computer]["computer-setup"] = update_to_v2_entry_points(final_dict_v2[domain][computer]["computer-setup"])
```

Comment:
I would rather run something like this here:
```suggestion
            update_to_v2_entry_points(final_dict_v2[domain][computer]["computer-setup"])
```
and remove `res = copy.deepcopy(comp_setup)` line in `update_to_v2_entry_points`. The dictionary is already| "deep copied", so this it is safe to modify it directly here.